effective-image-guide/readme.md

```diff
@@ -0,0 +1,56 @@
+>**The goal of this guide:** to explore what makes an image effective in relation to an article.
+
+# Sciencing the $@%* out of images
+
+When used correctly, effective visuals are extremely powerful to communicate an appropriate message. Let's look at some of the science behind visual content and what makes an image effective.
+
+## First off: Why visual content is so important
+
+Here are some telling stats as researched by [Buffer's](https://buffer.com/)  [Kevan Lee](https://blog.bufferapp.com/author/kevanlee) on just how big visual content has become and how it’s changing the way we all approach marketing:
```

Comment:
Because this section already goes outside the scope of the achievable, I took out all the specific stats and just linked to a few articles that already have them.